packages/react/src/_popover/Popover.tsx

```diff
@@ -28,11 +30,11 @@ export const Popover = forwardRefWithAs<'div', PopoverProps>(function Popover(
 			borderColor="muted"
 			rounded
 			css={{
-				position: 'relative',
-				overflowY: 'auto',
 				boxShadow: tokens.shadow.lg,
-				zIndex: tokens.zIndex.popover,
+				overflow: 'auto',
```

Comment:
Just checking that this `overflow` is required?

Reply:
This one makes sense to me. More so than `overflowY`. I don't fully understand why, but `overflowY` enables scrolling along the x-axis as well. So, this is more obvious WRT the resulting behaviour. I could be missing something, mind you. The other one you mentioned below isn't needed.